physionet-django/user/models.py

```diff
@@ -1164,3 +1164,33 @@ class CodeOfConductSignature(models.Model):
 
     class Meta:
         default_permissions = ()
+
+
+class Event(models.Model):
+    """
+    Captures information on events such as datathons, workshops and classes.
+    Used to allow event hosts to assist with credentialing.
+    """
+    name = models.TextField()
+    event_type = models.PositiveSmallIntegerField(choices=EventType.choices(), default=EventType.COURSE)
+    host = models.ForeignKey(User, on_delete=models.CASCADE)
+    added_datetime = models.DateTimeField(auto_now_add=True)
+    start_datetime = models.DateTimeField()
+    end_datetime = models.DateTimeField()
+
+    class Meta:
+        unique_together = ('name', 'host')
+
+
+class Participants(models.Model):
+    """
+    Captures information about participants in an event.
+    """
+    participant = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
update to match the model in your current branch (something like:
`user = models.ForeignKey(User, on_delete=models.CASCADE, related_name = "participants"`)

as we are considering a new app for events, you could hold off making updates to this pull request until we decide exactly what we want..